fenwick/src/main/java/org/opencadc/fenwick/AllArtifacts.java

```diff
@@ -80,4 +84,15 @@
     public AllArtifacts() {
         super();
     }
+
+    /**
+     * Obtain the iterator of clauses used to build a query to include the Artifacts being merged.
+     *
+     * @return An empty iterator.
+     */
+    @Override
```

Comment:
Unless we need multiple iterator-returning methods, just name this one `iterator()`